src/microbe_stage/editor/MicrobeEditorAction.cs

```diff
@@ -29,10 +32,11 @@ public class MicrobeEditorAction : ReversableAction
 
     public MicrobeEditorAction(MicrobeEditor editor, int cost,
         Action<MicrobeEditorAction> redo,
-        Action<MicrobeEditorAction> undo, IMicrobeEditorActionData data = null)
+        Action<MicrobeEditorAction> undo, string name, IMicrobeEditorActionData data = null)
```

Comment:
I don't like that a name was added here. If this kind of identification is needed, an enum should be created. However, I don't think we need to do that yet, as we can kinda dirtily detect based on the class that Data is which action this is.